### DISCUSSION TITLE
AlDente support on Windows PC

### DISCUSSION THREAD

**Discussion by chips88** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#632] [General]
I came across AlDente but disappointed to find out that it only runs on MacBook or Windows running on VM but still on MacBook. I have a Windows PC. Is there plan in the future roadmap to support Windows PC? If not, why?

**Comment by MatthiasKerbl** [comment]
> I came across AlDente but disappointed to find out that it only runs on MacBook or Windows running on VM but still on MacBook. I have a Windows PC. Is there plan in the future roadmap to support Windows PC? If not, why?

Hi @chips88,

Really good question.
The reason AlDente is only available on MacBooks is that it is really difficult to develop AlDente further to include windows PCs. We would need to find a different solution for almost every PC manufacturer if it even is possible. Since we already have a lot of development work to do with "only" MacBooks, it is not really possible to expand to PC as well at this moment.

Hope you understand our situation.

Cheers,
Matthias
